lib/checks/require/all_checks.rb

```diff
@@ -0,0 +1,10 @@
+module Checks
+  class Require
+
+    private_class_method :initialize
```

Comment:
I think here you can just make `Require` a module instead of a class, rather than make `initialize` private.

i.e.

```ruby
module Checks
  module Require
    def self.all_checks
    end
  end
end
```